packages/devtools_server/lib/src/server.dart

```diff
@@ -120,6 +128,7 @@ Future<HttpServer> serveDevToolsWithArgs(List<String> arguments,
   final bool enableNotifications = args[argEnableNotifications];
   final port = args[argPort] != null ? int.tryParse(args[argPort]) ?? 0 : 0;
   final bool headlessMode = args[argHeadlessMode];
+  final bool debugMode = args['debug'];
```

Comment:
perhaps make `debug` a constant like the other keys (e.g. argHeadlessMode)

Reply:
will do